salt/modules/salt_check.py

```diff
@@ -0,0 +1,449 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+'''This custom salt module makes it easy to test salt states and highstates.
+   Author: William Cannon  william period cannon at gmail dot com'''
+import os
+import os.path
+import yaml
+import salt.client
+import salt.minion
+import salt.config
+
+
+class SaltCheck(object):
+    '''
+    This class implements the salt_check
+    '''
+
+    def __init__(self):
+        self.__opts__ = salt.config.minion_config('/etc/salt/minion')
```

Comment:
@thatch45 Probably not in an instance method like this. Though, that *might* have worked at one time. I forget exactly.